share/man/man7/networking.7

```diff
@@ -0,0 +1,59 @@
+.\"-
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.\" Copyright (c) 2023 Alexander Ziaee
+.\"
+.\" Redistribution and use in source and binary forms, with or without
+.\" modification, are permitted provided that the following conditions
+.\" are met:
+.\" 1. Redistributions of source code must retain the above copyright
+.\"    notice, this list of conditions and the following disclaimer.
+.\" 2. Redistributions in binary form must reproduce the above copyright
+.\"    notice, this list of conditions and the following disclaimer in the
+.\"    documentation and/or other materials provided with the distribution.
+.\"
+.\" THIS SOFTWARE IS PROVIDED BY THE AUTHORS AND CONTRIBUTORS ``AS IS'' AND
+.\" ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+.\" IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+.\" ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE
+.\" FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+.\" DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+.\" OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+.\" HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+.\" LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+.\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+.\" SUCH DAMAGE.
+.\"
+.Dd August 29, 2023
+.Dt "NETWORKING" 7
+.Os
+.Sh NAME
+.Nm networking ,
+.Nm wifi
+.Nd quickstart guide to connecting to a network
+.Sh EXAMPLES
+.Ss Connecting to an ethernet network with DHCP:
+.Dl # dhclient em0
+.Ss Connecting to a wifi network:
+Identify your wifi hardware:
+.Dl % sysctl net.wlan.devices
+Configure your wifi hardware as wlan0 interface:
+.Dl # sysrc wlans_iwn0="wlan0"
+Set that interface to negotiate a dhcp lease with wpa supplicant:
+.Dl # sysrc ifconfig_wlan0="WPA SYNCDHCP"
+Enter the wifi network's details:
+.Dl # echo "network={ssid="myssid" psk="mypsk"} >> wpa_supplicant.conf
+Restart the network interface daemon:
+.Dl # service netif restart
+.Ss Scanning for wifi networks:
```

Comment:
```suggestion
.Ss Connecting to a Wi-Fi network:
Identify your Wi-Fi hardware:
.Dl % sysctl net.wlan.devices
Configure your Wi-Fi hardware as wlan0 interface:
.Dl # sysrc wlans_iwn0="wlan0"
Set that interface to negotiate a dhcp lease with wpa supplicant:
.Dl # sysrc ifconfig_wlan0="WPA SYNCDHCP"
Enter the details of the Wi-Fi network:
.Dl # echo "network={ssid="myssid" psk="mypsk"} >> wpa_supplicant.conf
Restart the network interface daemon:
.Dl # service netif restart
.Ss Scanning for Wi-Fi networks:
```

Probably more accurate to say _Wi-Fi_.

```text
% rg --count --sort path -e 'Wi-Fi' /usr/doc
/usr/doc/documentation/content/de/books/handbook/advanced-networking/_index.adoc:2
/usr/doc/documentation/content/el/books/handbook/advanced-networking/_index.adoc:3
/usr/doc/documentation/content/en/books/handbook/advanced-networking/_index.adoc:2
/usr/doc/documentation/content/en/books/handbook/advanced-networking/_index.po:2
/usr/doc/documentation/content/es/books/handbook/advanced-networking/_index.adoc:2
/usr/doc/documentation/content/es/books/handbook/advanced-networking/_index.po:6
/usr/doc/documentation/content/fr/books/handbook/advanced-networking/_index.adoc:1
/usr/doc/documentation/content/hu/books/handbook/advanced-networking/_index.adoc:1
/usr/doc/documentation/content/ja/books/handbook/advanced-networking/_index.adoc:1
/usr/doc/documentation/content/mn/books/handbook/advanced-networking/_index.adoc:2
/usr/doc/documentation/content/nl/books/handbook/advanced-networking/_index.adoc:3
/usr/doc/documentation/content/pl/books/handbook/advanced-networking/_index.adoc:3
/usr/doc/documentation/content/pt-br/books/handbook/advanced-networking/_index.adoc:2
/usr/doc/documentation/content/ru/books/handbook/advanced-networking/_index.adoc:1
/usr/doc/documentation/content/zh-cn/books/handbook/advanced-networking/_index.adoc:3
/usr/doc/documentation/content/zh-tw/books/handbook/advanced-networking/_index.adoc:3
/usr/doc/website/content/en/releases/12.4R/errata.adoc:1
/usr/doc/website/content/en/releases/13.2R/errata.adoc:1
/usr/doc/website/content/en/status/report-2017-07-2017-09.html:1
/usr/doc/website/content/en/status/report-2018-01-2018-09.html:1
/usr/doc/website/content/en/status/report-2022-07-2022-09/wtap.adoc:1
/usr/doc/website/content/en/status/report-2023-04-2023-06/batman.adoc:1
/usr/doc/website/data/en/events/events2022.toml:1
/usr/doc/website/static/security/advisories/FreeBSD-EN-22:18.wifi.asc:1
/usr/doc/website/static/security/advisories/FreeBSD-SA-17:07.wpa.asc:4
/usr/doc/website/static/security/advisories/FreeBSD-SA-19:03.wpa.asc:2
/usr/doc/website/static/security/advisories/FreeBSD-SA-22:02.wifi.asc:2
/usr/doc/website/static/security/advisories/FreeBSD-SA-22:07.wifi_meshid.asc:3
/usr/doc/website/static/security/advisories/FreeBSD-SA-23:11.wifi.asc:4
/usr/doc/website/static/security/patches/SA-19:03/wpa-11.patch:44
/usr/doc/website/static/security/patches/SA-19:03/wpa-12.patch:39
% 
```

Avoid contractions. 

<https://docs.freebsd.org/en/books/fdp-primer/book/#writing-style-guidelines>

Thanks